File: android/src/main/java/com/audiorecorderplayer/nitro/NitroAudioRecorderPlayerOnLoad.kt
package com.audiorecorderplayer.nitro

import com.facebook.react.bridge.ReactApplicationContext
import com.margelo.nitro.NitroModules

class NitroAudioRecorderPlayerOnLoad {
    companion object {
        @JvmStatic
        fun load(context: ReactApplicationContext) {
            NitroModules.register(HybridAudioRecorderPlayer::class.java) { 
                HybridAudioRecorderPlayer(context)
            }
        }
    }
}

File: ios/NitroAudioRecorderPlayerOnLoad.swift
import Foundation
import NitroModules

@objc(NitroAudioRecorderPlayerOnLoad)
public class NitroAudioRecorderPlayerOnLoad: NSObject {
    @objc
    public static func load() {
        NitroModules.register(HybridAudioRecorderPlayer.self)
    }
}